testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/migration/MigrationTest.java

```diff
@@ -58,18 +59,28 @@ public void beforeMigrationTest() {
         addTestRealmToTestRealmReps(migrationRealm2);
         addTestRealmToTestRealmReps(migrationRealm3);
     }
+
     private void addTestRealmToTestRealmReps(RealmResource realm) {
         try {
             testRealmReps.add(realm.toRepresentation());
         } catch (NotFoundException ex) {
         }
     }
+
+    @Test
+    @Migration(versionFrom = "3.4.3.Final")
+    public void migration3_4_3Test() {
+        testMigratedData();
+        testMigrationTo4_x();
+    }
+
     @Test
     @Migration(versionFrom = "2.5.5.Final")
     public void migration2_5_5Test() {
         testMigratedData();
-        testMigrationTo3_x_and_higher();
+        testMigrationTo3_x();
```

Comment:
It seems that until now, we tested migration from 2.5.5 to all the versions (including newest "testMigrationTo4_x"). With this change, we will test from 2.5.5 just to 3.x, but not to latest 4.x if I understand correctly. Is this the change we want to introduce? Shouldn't we always test to newest version?